### PR TITLE
[OSDOCS#9302]: Fixes for OSDOCS-9302

### PR DESCRIPTION
**Pull request by obrown1205**
Fixes to https://github.com/openshift/openshift-docs/pull/70182

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9302

Link to docs preview:
https://70637--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-CAPO-tech-preview-no-upgrade

QE review:
N/A

Additional information:

### CODE REVIEW

**Comment by mjpytlak** [comment]
Thanks for adding the heading @obrown1205

**Comment by mramendi** [comment]
@obrown1205 I don't see a SME or QE ack, but the Jira does not look like it's just doc maintenance. Please provide at least QE ack before peer review

File: modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
```
Comment (mramendi) [inline modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc:1]: This module does not appear to be included in the ocp-4-15-release-notes assembly. Adding an orphan module to the repository is to be avoided if at all possible. As the same information appears to be added to the assembly itself, I would suggest removing this file.

**Comment by mramendi** [comment]
@obrown1205 please ensure this also gets QE signoff (and SME if necessary) before seeking to merge. Also, it's general practice to state a direct link to the preview in the description. In this case the link is https://70637--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-installation-and-update

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -58,14 +58,10 @@ This release adds improvements related to the following components and concepts.
...
+[discrete]
+=== CAPO integration into the cluster CAPI Operator (Tech Preview)
 
-For more information, see xref:../installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc#user-managed-encryption-ibm-cloud[User-managed encryption for IBM Cloud].
+The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the Coherent Accelerator Processor Interface (CAPI) Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional {product-title} cluster are automatically created for the current {product-title} cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:64]: There is a lot of passive voice here. In my view this should use the active voice. Something like:

If you enable the `TechPreviewNoUpgrade` feature flag, the Coherent Accelerator Processor Interface (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.

The part about configuring the CAPI machine should be a separate paragraph and also use the active voice: "You can now configure..."

The "equivalent but not identical" sentence needs to be expanded in my opinion as right now it is very hard to guess the meaning (I could not). It should probably be a separate paragraph too.

**Review by mramendi — COMMENTED** [review]
As I am in the peer review induction process @JoeAldinger will finish the review

File: modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc (hunk 1 above)
Comment (JoeAldinger) [reply modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc:1]: Release notes are weird. I see why you've included this @obrown1205 but @mramendi is correct this just needs to be deleted.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -58,14 +58,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-installation-and-update"]
 === Installation and update
 
-The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the CAPI Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional OpenShift cluster are automatically created for the current OpenShift cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-15-release-notes.adoc:61]: Ok, so it took me a minute to follow the breadcrumbs. I _think_ what you want to do is add `[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
== CAPO integration into the cluster CAPI Operator(Tech Preview)` to line 60 of the RN file (i.e. this file). 

You then need to put back line 63-68. I'm guessing you had a merge conflict and just picked the wrong content. If you need help sorting this ping me @obrown1205

**Review by JoeAldinger — COMMENTED** [review]
Need to fix some stuff and I like @mramendi suggestions for sharpening content if you'd like.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 3 above)
Comment (mjpytlak) [reply release_notes/ocp-4-15-release-notes.adoc:61]: Thanks for spotting this @JoeAldinger. Yes as it stands this PR would remove my release note for user-managed encryption.

**Comment by obrown1205** [comment]
> @obrown1205 please ensure this also gets QE signoff (and SME if necessary) before seeking to merge. Also, it's general practice to state a direct link to the preview in the description. In this case the link is https://70637--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-installation-and-update

Thank you! Technically this PR had been peer-reviewed already (see the first line's link to the previous PR: https://github.com/openshift/openshift-docs/pull/70182) and the release note does not need QE review. I did receive SME review there as well. I appreciate your feedback and will apply!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 3 above)
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc:61]: Thank you!

File: modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc (hunk 1 above)
Comment (obrown1205) [reply modules/ocp-4-15-CAPO-tech-preview-no-upgrade.adoc]: Done :)

File: release_notes/ocp-4-15-release-notes.adoc (hunk 2 above)
Comment (obrown1205) [reply release_notes/ocp-4-15-release-notes.adoc]: Thank you for the feedback - I've adjusted :) the last sentence relates to the resources of CAPI vs MAPI. I've rephrased a bit.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -58,7 +58,12 @@ This release adds improvements related to the following components and concepts.
...
+[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
+==== CAPO integration into the cluster CAPI Operator (Tech Preview)
+
+If you enable the `TechPreviewNoUpgrade` feature flag, the Coherent Accelerator Processor Interface (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
```
Comment (dulek) [inline release_notes/ocp-4-15-release-notes.adoc:64]: CAPI doesn't stand for "Coherent Accelerator Processor Interface", it's just Cluster API.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 5]
```diff
@@ -58,7 +58,12 @@ This release adds improvements related to the following components and concepts.
...
+
+It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources and not identical.
```
Comment (dulek) [inline release_notes/ocp-4-15-release-notes.adoc:66]: MAPI doesn't stand for "Mail Application Programming Interface", it's just Machine API.

**Review by dulek — COMMENTED** [review]
Hey @obrown1205, I pointed out two issues with acronyms here. Can you please create a PR to fix them? Otherwise these sentences are very confusing from the technical standpoint.

**Pull request merged**